examples/experimental/i3s-17-and-debug/constants.js

```diff
@@ -45,4 +45,9 @@ export const BOUNDING_VOLUME_TYPE = {
   OBB: ORIENTED_BOUNDING_BOX
 };
 
+export const BOUNDING_VOLUME_MESH_TYPE = {
```

Comment:
Do we need a separate type for this?

Reply:
I fell that existing bounding volume types are not good as object keys for tile.userData object.
```
export const BOUNDING_SPHERE = 'Bounding Sphere';
export const ORIENTED_BOUNDING_BOX = 'Oriented Bounding Box';
```

Reply:
I see. You could also make the types more general, and have a separate map for type names

```
enum BOUNDING_VOLUME_TYPE {
  BOUNDING_SHPERE,
  ORIENTED_BOUNDING_BOX
}

const BOUNDING_VOLUME_NAME = {
  [BOUNDING_VOLUME_TYPE.BOUNDING_SPHERE]: 'Bounding Sphere',
...
}
```

Reply:
Will do in further PR